### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by trustin**
- Armeria 0.98.1 -> 0.98.4
- Brave 5.9.4 -> 5.9.5

### CODE REVIEW

**Comment by trustin** [comment]
Will need 0.99.0 to upgrade Brave to 5.10.1.

File: pom.xml [hunk 1]
```diff
@@ -36,14 +36,14 @@
...
-        <version>0.98.1</version>
+        <version>0.98.4</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
       <dependency>
         <groupId>io.zipkin.brave</groupId>
         <artifactId>brave-bom</artifactId>
-        <version>5.9.4</version>
+        <version>5.9.5</version>
```
Comment (adriancole) [inline pom.xml]: might as well 5.10.1 I think?

```suggestion
        <version>5.10.1</version>
```

Comment (trustin) [reply pom.xml]: No need to wait for 5.10.1?

Comment (trustin) [reply pom.xml]: No need to wait for 0.99.0?

Comment (adriancole) [reply pom.xml]: nope we don't break anything.. we'll be 5.x for the foreseeable future and won't remove any deprecated things

Comment (adriancole) [reply pom.xml]: ex in sleuth, we always release latest brave, even on old branches

**Pull request merged**